CONTRIBUTING.md

```diff
@@ -1,8 +1,21 @@
 Thanks for taking the time to make the osu! wiki better!
 
-To help you get started on the right track, check out the following resources to help you get started:
+Anyone is allowed (and encouraged!) to contribute to the osu! wiki.
 
-- The [Article Styling Criteria](https://osu.ppy.sh/help/wiki/Article_Styling_Criteria) for the styling standards in wiki.
+## Getting Started
 
-- The [Getting Started Guide](https://osu.ppy.sh/help/wiki/osu!wiki_contribution_guide) _(New to the GitHub workflow? You should read this first)_.
+### Markdown
 
+The osu! wiki is formatted using Markdown (specifically [GitHub-Flavoured Markdown (GFM)](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/ "GitHub Help: Writing on GitHub")), a markup language that allows for text to have basic styling.
+
+[adam-p's Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) is a fantastic tool of reference if you need to quickly learn about or refresh your memory of a Markdown feature.
+
+### Git and GitHub
+
+Since the osu! wiki is managed through this git repository, you'll need some basic knowledge of git in order to contribute.
```

Comment:
```suggestion
Since the osu! wiki is managed through this GitHub repository, you'll need some basic knowledge of [Git](https://git-scm.com/) in order to contribute.
```